File: day 32 python/homework/hw8.py
# 14) while ციკლით დაამატეთ ცარიელ სეტში 1- დან 20 - მდე რიცხვებ(update - ის გამოყენებით).

nums = set()
result = []
x = 0
y = 20
while x != y:
    x+=1
    result.append(x)
nums.update(result)
print(nums) 

File: day 55 Js/homework/hw8.js
// დაწერეთ კოდი, რომელიც აიღებს მასივს [4,9,16,25], თითოეული რიცხვიდან გამოიტანოს მისი კვადრატული ფესვი

let nums = [4,9,16,25];
let result;

for (let i = 0; i < nums.length; i++){
    console.log(`${nums[i]} = ${result = Math.sqrt(nums[i])}`)

}